### DISCUSSION TITLE
What are the brand guidelines on using the LinuxGSM name or logo?

### DISCUSSION THREAD

**Discussion by LoadingStill** [GameServerManagers/LinuxGSM#4371] [General]
I see the project is under the MIT License, but I want to make sure my project is not going to interfere on any trademarks or go against the project's wishes.  Does LinuxGSM have a Brand Guideline for how to use its name and or logo?  Here is an example of what I am referring to the Jet Brains Brand Guidelines https://www.jetbrains.com/company/brand/#brand-guidelines  
I have looked for an answer for this on the discussion board, but I did not find an answer to this exact question, if I just missed it I apologize for the duplicate question.

Edit was changing the link, it was not working for me after post. && spelling

**Comment by h3o66** [comment]
There are a few branding logos here:
https://docs.linuxgsm.com/other/brand

for everything else @dgibbs64 can probably say more.
